src/main/java/teammates/common/util/Config.java

```diff
@@ -39,6 +39,12 @@
     
     /** The value of the "app.student.motd.url" in build.properties file */
     public static String STUDENT_MOTD_URL;
+    
+    /** The value of the "app.sendgrid.username" in build.properties file */
+    public static String SENDGRID_USERNAME;
+    
+    /** The value of the "app.sendgrid.password" in build.properties file */
+    public static String SENDGRID_PASSWORD;
```

Comment:
Optionally, they can use sendgrid, especially if their fixes relate to
email sending. They should use their own account.